20200821/MINUTES.md

```diff
@@ -0,0 +1,168 @@
+# Minutes from the Sardana Follow-up Meeting - 2020/08/21
+
+Held on 2020/08/21 at 11:00
+
+Participants: ALBA - Zbigniew Reszela, DESY - Teresa Núñez, MAXIV - Aureo Freitas (Mirjam Lindberg, David Erb), Solaris - Michal Piekarski 
+
+1. Urgent user problems/issues - Round table
+
+Alba:
+- a fourth beamline finished the migration to python3, not tried with users
+  due to reduce operation, it will start after summer shutdown
+- one beamline was updated to the last python2 release and found a problem
+  related to the MacroButton.
+- bug 1351, it happens in loop of movements whe one motor has backlash, the
+  backlash movement crash with the next motor in the loop, there is
+  no lock protection for backlash movements. Workaround: put the motor
+  with backlash (if only one) at the end of the loop. A fix has to be done.
+- documentation about the diferent ways of interacting with Tango devices from
+  the macros was put in Sardana FAQ due to the confusion of some users.
+- it was requested the possibility of modifying the ScanUser environment
+  variable, a PR (#1355) was created allowing that.
+- a discussion was started (PR #1356 - WIP) about adding missing epoch to
+  spec recorder.
+- PR sent by Tiago (#1367) for solving the problem with functions that
+  were not taken by the MacroServer.
+- PR (#1386) for adding scan widgets was sent.
+- PR (#1355) for adding custom data to the scan was sent.
+- added the possibility of adding custom data to the scan files (using
+  the function addCustomData), the way used by spec was looked at.
+
+MaXIV:
+- they are testing python3 with the last release, they found some problems with
+  python but the main problem was with nexus5. They manages to solve the
+  problems using singularity and dockers and now they are ready. They have
+  still problems with controllers talking directly to the hardware.
+  Next week they want to deploy sardana3 for one beamline.
+  The main issue is with controllers and macros developed by the users.
+  David is working in two new controllers talking directly to the
+  hardware. Some problems trying to understand the behaviour of the
+  TriggerGateController, it does not work with the time scan.
+
+Solaris:
+- issues about Sardana customization. It was asked for an easy way to set
+  the logging level (they want to set it to INFO) and about naming templates
+  (the default ones do not follow their naming convention and renaming by hand
+  is a lot of work). Zibi suggested to use the Sardana custom
+  settings. In solaris they use a sardana config file and also define there
```

Comment:
I think it was Aureo who mentioned that they use `dsconfig` to easily rename Sardana elements.